plugins/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(
+  IRODS_HTTP_PLUGINS_DIRECTORY
+  "/usr/lib/irods_http_api/plugins"
```

Comment:
@SwooshyCueb Do you have any guidance for this? Would taking the `InstallDirs.cmake` from iRODS be the best way forward?

Reply:
We don't need to copy `InstallDirs.cmake`. We can just use `CMAKE_INSTALL_LIBDIR` instead of `/usr/lib`. Make sure [`GNUInstallDirs`](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) is included.